mmv1/products/certificatemanager/Certificate.yaml

```diff
@@ -108,7 +109,9 @@ properties:
 
       EDGE_CACHE: Certificates with scope EDGE_CACHE are special-purposed certificates,
       served from non-core Google data centers.
-      Currently allowed only for managed certificates.
+
+      ALL_REGIONS: Certificates with ALL_REGIONS scope are served from all GCP regions (You can only use ALL_REGIONS with global certs).
```

Comment:
Also, the `scope` is defined as an enum in the api page, but It's a sting in TF. is changing it from string to ENUM going to be a breaking a change?

Reply:
Yes, changing a string to an enum would be a breaking change.